dracut/30ignition/module-setup.sh

```diff
@@ -29,32 +29,12 @@ install() {
     inst_script "$moddir/ignition-setup.sh" \
         "/usr/sbin/ignition-setup"
 
-#   inst_script "$moddir/retry-umount.sh" \
-#       "/usr/sbin/retry-umount"
-
     inst_simple "$moddir/ignition-generator" \
         "$systemdutildir/system-generators/ignition-generator"
 
-    inst_simple "$moddir/ignition-disks.service" \
-        "$systemdsystemunitdir/ignition-disks.service"
-
-    inst_simple "$moddir/ignition-files.service" \
-        "$systemdsystemunitdir/ignition-files.service"
-
-    inst_simple "$moddir/ignition-ask-var-mount.service" \
-        "$systemdsystemunitdir/ignition-ask-var-mount.service"
-
-    inst_simple "$moddir/ignition-remount-sysroot.service" \
-        "$systemdutildir/system/ignition-remount-sysroot.service"
-
-#   inst_simple "$moddir/sysroot-boot.service" \
-#       "$systemdsystemunitdir/sysroot-boot.service"
-
-#   inst_simple "$moddir/coreos-digitalocean-network.service" \
-#       "$systemdsystemunitdir/coreos-digitalocean-network.service"
-
-#   inst_simple "$moddir/coreos-static-network.service" \
-#       "$systemdsystemunitdir/coreos-static-network.service"
+    for svc in "$moddir"/*.service; do
+        inst_simple "$svc" "$systemdsystemunitdir/$(basename $svc)"
+    done
```

Comment:
This is mostly just me but I kind of like being explicit here. Since there are only a few I would prefer naming them all like we were doing before.

Reply:
The context for this change is that #47 adds two more units. :) I don't mind reverting back though if folks prefer.